thunderbolt/client/gcs_client.py

```diff
@@ -42,3 +42,11 @@ def _get_gcs_object_info(self, x: str) -> Dict[str, str]:
         """get GCS object meta data"""
         bucket, obj = self.gcs_client._path_to_bucket_and_key(x)
         return self.gcs_client.client.objects().get(bucket=bucket, object=obj).execute()
+
+    def convert_absolute_path(self, x: str) -> str:
```

Comment:
[IMO] <convert "to" absolute path> might be good name preventing confusing between "to" and "from".